development/Catch2/Catch2.SlackBuild

```diff
@@ -63,6 +63,14 @@ else
   LIBDIRSUFFIX=""
 fi
 
+#Test for multilib
+COMPAT32=${COMPAT32@L}
```

Comment:
Not sure what the point of allowing this to be passed in through an ENV var if we are going to ignore it and overwrite it anyway?